eotsmanager/cmd/eotsd/daemon/pop_test.go

```diff
@@ -0,0 +1,67 @@
+package daemon_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"path/filepath"
+	"testing"
+
+	"github.com/babylonchain/babylon/testutil/datagen"
+	bbn "github.com/babylonchain/babylon/types"
+	dcli "github.com/babylonchain/finality-provider/eotsmanager/cmd/eotsd/daemon"
+	"github.com/babylonchain/finality-provider/testutil"
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/stretchr/testify/require"
+	"github.com/urfave/cli"
+)
+
+func FuzzPoPExport(f *testing.F) {
+	testutil.AddRandomSeedsToFuzzer(f, 10)
+	f.Fuzz(func(t *testing.T, seed int64) {
+		r := rand.New(rand.NewSource(seed))
+
+		tempDir := t.TempDir()
+		homeDir := filepath.Join(tempDir, "eots-home")
+		app := testApp()
+
+		// init config in home folder
+		hFlag := fmt.Sprintf("--home=%s", homeDir)
+		err := app.Run([]string{"eotsd", "init", hFlag})
+		require.NoError(t, err)
+
+		keyName := testutil.GenRandomHexStr(r, 10)
+		keyNameFlag := fmt.Sprintf("--key-name=%s", keyName)
+
+		outputKeysAdd := appRunWithOutput(r, t, app, []string{"eotsd", "keys", "add", hFlag, keyNameFlag})
+		keyOutJson := searchInTxt(outputKeysAdd, "for recovery):")
+
+		var keyOut dcli.KeyOutput
+		err = json.Unmarshal([]byte(keyOutJson), &keyOut)
+		require.NoError(t, err)
+
+		bbnAddr := datagen.GenRandomAccount().GetAddress()
+
+		btcPkFlag := fmt.Sprintf("--btc-pk=%s", keyOut.PubKeyHex)
+		exportedPoP := appRunPoPExport(r, t, app, []string{bbnAddr.String(), hFlag, btcPkFlag})
+		require.NotNil(t, exportedPoP)
+		require.NoError(t, exportedPoP.PoP.ValidateBasic())
+
+		btcPubKey, err := bbn.NewBIP340PubKeyFromHex(exportedPoP.PubKeyHex)
+		require.NoError(t, err)
+		require.NoError(t, exportedPoP.PoP.Verify(bbnAddr, btcPubKey, &chaincfg.MainNetParams))
```

Comment:
```suggestion
		require.NoError(t, exportedPoP.PoP.Verify(bbnAddr, btcPubKey, &chaincfg.SigNetParams))
```

We use `signet` in default but here btc net params does not matter. Sill better to keep it consistent

Reply:
https://github.com/babylonchain/finality-provider/pull/460/commits/d1c66761d2fbe7d9ca9ab7ddb472ccb3db28cc26